modules/support.adoc

```diff
@@ -5,26 +5,11 @@
 [id="support_{context}"]
 = Getting support
 
-If you experience difficulty with a procedure described in this documentation,
-visit the link:http://access.redhat.com[Red Hat Customer Portal]. Through the
-Customer Portal, you can:
+If you experience difficulty with a procedure described in this documentation, or with OpenShift in general, visit the link:http://access.redhat.com[Red Hat Customer Portal]. Through the Customer Portal, you can:
```

Comment:
{product-title}
I suggest a find/replace for instances of `OpenShift` - I'll stop calling it out now. 😃

Reply:
s/Through/From/

Reply:
Resolved.